server/flaskr/blueprints/decision.py

```
@@ -45,3 +38,19 @@ def get_decisions():
+@decisions_bp.patch("/<string:id>/status")
+@input_middleware(LambdaBuilder(("status", EnumValidator(DecisionStatus))))
+def patch_meeting_agenda_status(status, id: str):
+    try:
+        result = False
+        if status == 'completed':
+            result = DecisionDataHandler.complete_decision(id)
+        elif status == 'cancelled':
+            result = DecisionDataHandler.cancel_decision(id)
+        if not result:
+            return '', 404
+        return '', 204
```

Ruff (0.15.5)

[warning] Consider moving this statement to an `else` block

(TRY300)